src/zfm20/zfm20.cxx

```diff
@@ -540,10 +439,10 @@ uint8_t ZFM20::deleteDB()
   return rPkt[9];
 }
 
-uint8_t ZFM20::search(int slot, uint16_t *id, uint16_t *score)
+uint8_t ZFM20::search(int slot, uint16_t &id, uint16_t &score)
 {
```

Comment:
Have you verified the device with this?  I always avoided exposing references to SWIG, due to [this SWIG documentation](http://www.swig.org/Doc1.3/SWIGPlus.html#SWIGPlus_nn18)

It says references are just converted to pointers and there are apparently issues with const'ness (due to the fact that in SWIG they are converted to pointers).  But if the device still works ok in all languages, go for it.

But, if it works...